lib/component-names.js

```diff
@@ -11,7 +12,12 @@ module.exports = {
     if (actualPath.includes(classicStyleDir)) {
       terminator = '.';
       pathSegementToRemove = 'styles/' + classicStyleDir + '/';
-    }
+    } else { // Colocated component stylesheet
```

Comment:
I think this is where the failing tests aren't passing. If there was a way to only have this branch be exercised if it's colocated styles, that would be beneficial.

Reply:
The failing tests are due to other octane changes on my second viewing now.